lib/mutant/reporter/cli/printer/subject_result.rb

```diff
@@ -7,7 +7,7 @@ class Printer
         # Subject result printer
         class SubjectResult < self
 
-          delegate :subject, :alive_mutation_results, :tests
+          delegate :subject, :uncovered_results, :tests
```

Comment:
I'm slowly moving the wording away from "alive" and "killed" to "uncovered" and "covered".